exchanges/bitfinex.js

```diff
@@ -39,7 +39,7 @@ var retryForever = {
 };
 
 // Probably we need to update these string
-var recoverableErrors = new RegExp(/(SOCKETTIMEDOUT|TIMEDOUT|CONNRESET|CONNREFUSED|NOTFOUND|StatusCodeError: 429|StatusCodeError: 5)/)
+var recoverableErrors = new RegExp(/(SOCKETTIMEDOUT|TIMEDOUT|CONNRESET|CONNREFUSED|NOTFOUND|429|443|5\d\d)/g);
```

Comment:
Is this supposed to be part of this PR?

Reply:
I’m sorry, this will update the Bitfinex string. Maybe I’ll submit another PR.

Reply:
Maybe it needs to be updated, did you add this specifically because it caught errors better?

Reply:
Yes, this string caught erros better.It can be committed.